core/src/main/resources/application.properties

```
@@ -51,21 +51,30 @@ klaw.enable.authorization.ad=false
 
 # sso config
 klaw.enable.sso=false
-
 # Uncomment the below OAuth2 configuration to enable Google based authentication
+#spring.security.oauth2.client.registration.google.imageURI=assets/images/clients/google.svg
```

```suggestion
#klaw.security.oauth2.client.registration.google.imageURI=assets/images/clients/google.svg
```

As imageURI is a custom attribute, prefer to be it under klaw.